src/components/PostPageDetail/index.js

```diff
@@ -556,20 +582,6 @@ const PostPageDetailIdComponent = (props) => {
   const handleOnPressScore = () => {
     showScoreAlertDialog(item);
   };
-  const updateVoteLatestChildren = async (dataUpdated, data, level) => {
-    if (level === 3) {
-      const newComment = await updateVoteLatestChildrenLevel3(comments, dataUpdated);
-      saveComment(newComment, dispatchComment);
-    }
-    if (level === 1) {
-      const newComment = await updateVoteChildrenLevel1(comments, dataUpdated);
-      saveComment(newComment, dispatchComment);
-    }
-  };
-
-  const handlePaddingBottom = () => {
-    return comments.length <= 0 ? calculatePaddingBtm() : 0;
-  };
 
   const handleUpdateVote = () => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateVoteLatestChildren` function has been removed. If this function was being used elsewhere in the component, make sure its functionality is covered by other parts of the code.
```diff
- const updateVoteLatestChildren = async (dataUpdated, data, level) => {
-   if (level === 3) {
-     const newComment = await updateVoteLatestChildrenLevel3(comments, dataUpdated);
-     saveComment(newComment, dispatchComment);
-   }
-   if (level === 1) {
-     const newComment = await updateVoteChildrenLevel1(comments, dataUpdated);
-     saveComment(newComment, dispatchComment);
-   }
- };
+ const handleUpdateVote = () => {
+   getComment(false, true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->